plugins/inputs/win_perf_counters/win_perf_counters_test.go

```diff
@@ -435,6 +435,8 @@ func TestWinPerfcountersCollect1(t *testing.T) {
 	instances[0] = "_Total"
 	counters[0] = "Parking Status"
 
+	var expectedCounter string = "Parking_Status"
```

Comment:
These tests were broken as a part of this change set:

https://github.com/influxdata/telegraf/commit/015261a524e8f335f844e7e3ffdd75ea3b4f2b57